src/styles/main.scss

```
@@ -1,3 +1,55 @@
+@import "./variables";
+@import "./reset.scss";
+
 body {
-  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: (($cell-size + 1px) * 7) + 20px;
+  padding: 10px;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    border: 1px solid black;
+    text-align: center;
+    line-height: $cell-size;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+    transition-duration: 0.5s;
+  }
+}
+
+@for $i from 1 to 32 {
```

https://sass-lang.com/documentation/at-rules/control/for

```suggestion
@for $i from 1 through 31 {
```